modules/systemd-journal/systemd-journal-grammar.ym

```diff
@@ -104,15 +104,15 @@ source_systemd_journal_option
           {
             if (last_journal_reader_options->recv_time_zone)
               free(last_journal_reader_options->recv_time_zone);
-            last_journal_reader_options->recv_time_zone = strdup($3);
+            last_journal_reader_options->recv_time_zone = g_strdup($3);
```

Comment:
this should have been a setter function though. This already improves things, so I wouldn't insist, but if we touch this code, we might have added the setter function too.

Reply:
sure, will do, it was just something I saw during review and wanted it get fixed.